lib/redcord/attribute.rb

```diff
@@ -58,6 +58,11 @@ def ttl(duration)
       class_variable_set(:@@ttl, duration)
     end
 
+    sig { returns(String) }
```

Comment:
Why do you need to define this here? This is also defined in serializer

Reply:
I moved this from serializer.rb to attribute.rb 🤔 I guess serializer.rb  is a better place for this method after all. Will put it back lol